rpc/params/ssh.go

```diff
@@ -64,13 +64,10 @@ type SSHHostKeyRequestArg struct {
 	Hostname string `json:"hostname"`
 }
 
-// PublicSSHHostKeyResult returns the host key for the target hostname.
-// Additionally, it returns the controller's SSH jump server's host key.
-//
-// We return the jump server's host key as to SSH to this unit, clients MUST
-// jump through the controller.
+// PublicSSHHostKeyResult returns the authorized key for the target hostname.
+// Additionally, it returns the controller's jump server's authorized key.
 type PublicSSHHostKeyResult struct {
-	Error             *Error `json:"error,omitempty"`
-	HostKey           string `json:"host-key"`
-	JumpServerHostKey string `json:"jump-server-host-key"`
+	Error                   *Error `json:"error,omitempty"`
+	AuthorizedKey           string `json:"authorized-key"`
```

Comment:
What's the correct name here? I.e. why are we switching to `AuthorizedKey` over what we had originally?
When I read authorized keys, I think of the sshd file `authorized_keys` which lists the public keys of users that can connect. These are the public key portion of the host keys of the server and unit.

Reply:
I've explained in the other comment.